libpod/container.go

```diff
@@ -262,6 +263,12 @@ type ContainerConfig struct {
 	// File containing the conmon PID
 	ConmonPidFile string `json:"conmonPidFile,omitempty"`
 	// TODO log options for log drivers
+
+	PostConfigureNetNS bool `json:"postConfigureNetNS"`
```

Comment:
Can we drop the bool and make this the default? I really hate having two slightly different ways of configuring net namespaces

Reply:
I agree, as we discussed it.  I've left this version only because it seemed like a big change on its own that requires more testing.  If something doesn't work right now in PostConfigureNetNS that will affect only containers using the new userNS features.